wiki/Community/User_title/fr.md

```diff
@@ -32,7 +30,7 @@ La couleur du titre d'un utilisateur est la même que celle de l'utilisateur, qu
 
 ![Capture d'écran de la barre latérale du poste de Damnae sur l'ancien site web](img/star-shooter.png "Le titre \"Star Shooter\" se trouve au-dessus du nombre de posts de Damnae.")
 
-Sur les forums de [l'ancien site d'osu!](https://old.ppy.sh), les utilisateurs pouvaient obtenir des titres en atteignant certains seuils de nombre de posts, comme suit :
+Sur les forums de [l'ancien site d'osu!](https://old.ppy.sh), les utilisateurs pouvaient obtenir des titres en atteignant un certains seuils de nombre de posts, comme suit:
```

Comment:
wtf when do you put a space and when do you not i would love to know

Reply:
We always put a space in french, my bad